### PR TITLE
No migrations left behind, in CI

### LINKED ISSUE #1590: No migrations left behind, in CI
If you have a PR which changes the models but forgot to generate the migrations for it, it should break CI. 

See https://github.com/mozilla/normandy/issues/1588#issuecomment-431135260 for background.

### PR DESCRIPTION
**Pull request by peterbe**
Fixes #1590

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
     working_directory: ~/normandy
 
     environment:
-        COMPOSE_FILE: ci/docker-compose.yml
+      COMPOSE_FILE: ci/docker-compose.yml
```
Comment (peterbe) [inline .circleci/config.yml]: My vscode fixes yaml files automatically and although this is unrelated I don't think it hurts to correct.

Comment (mythmon) [reply .circleci/config.yml]: In the future it would be good to only include these kind of fixes to files you fixed already. This change is more correct so lets keep it, but if we're going to have automatic fixes for these kind of files, we should add them as rules to Therapist instead of ad-hoc changes.
